cloudsaga/scenarios/public_resources.py

```diff
@@ -21,7 +21,7 @@
 
 
 sts = boto3.client('sts')
-region = os.environ['AWS_REGION']
+region = os.environ.get('AWS_REGION', 'us-east-1')
```

Comment:
This resolves a default workflow bombing out when attempting `cloudsaga -h` if `AWS_REGION` is not set.